great-idea/index.html

```diff
@@ -14,10 +14,79 @@
   <![endif]-->
 </head>
 
-  <body>
+  <body class="container">
+    <header>
```

Comment:
Under most circumstances, adding a `class="container"` on the body is suffice.  There will be sometimes that you will need to have some elements outside of your `container` class and using a `<div class='container'>` would be more flexible here.